keanu-python/keanu/algorithm/optimization.py

```diff
@@ -44,7 +55,13 @@ def __init__(self, net, max_evaluations=None, relative_threshold=None, absolute_
 
 
 class NonGradientOptimizer(Optimizer):
-    def __init__(self, net, max_evaluations=None, bounds_range=None, initial_trust_region_radius=None, stopping_trust_region_radius=None):
+    def __init__(self,
+                 net: Union[Vertex, BayesNet],
```

Comment:
Keep the ordering the same as for `GradientOptimizer`, :`net: Union[BayesNet, Vertex],`